e2e/main_page.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+import { Selector } from 'testcafe'; // first import testcafe selectors
+
+export default class MainPage {
+    constructor () {
+        this.createGame = Selector('#create-game');
```

Comment:
Where can I find `Selector('#create-game')`?

Reply:
The actual element is in frontend/src/views/Main.vue. The `#` means it's an ID selector (check out [CSS Selectors](https://www.w3schools.com/CSSref/css_selectors.asp) for more info